src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests/EventHandlers/ProviderRejectedChangeOfProviderRequestEventHandlerTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AutoFixture;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NServiceBus;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.CommitmentsV2.TestHelpers;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.Encoding;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests.EventHandlers
+{
+    [TestFixture]
+    public class ProviderRejectedChangeOfProviderRequestEventHandlerTests
+    {
+        private ProviderRejectedChangeOfProviderRequestEventHandlerTestFixture _fixture;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _fixture = new ProviderRejectedChangeOfProviderRequestEventHandlerTestFixture();
+        }
+
+        [Test]
+        public async Task WhenHandlingEvent_AndChangeOfPartyTypeIsChangeOfProvider_ThenSendEmailCommandShouldBeSent()
```

Comment:
may be a test to check if it is not change of provider then don't send the email